sdk/core/azure_core_amqp/src/fe2o3/messaging/mod.rs

```diff
@@ -712,7 +712,7 @@ mod tests {
             >::new(
                 seq.into_iter()
                     .map(|x| {
-                        let iter = x.into_iter().map(Into::into);
+                        let iter = x.into_iter();
```

Comment:
Clippy was complaining about this being unnecessary. If the tests pass with this change, I'm assuming it's correct.

> error: useless conversion to the same type: `serde_amqp::Value`
   --> sdk/core/azure_core_amqp/src/fe2o3/messaging/mod.rs:715:49
    |
715 |                         let iter = x.into_iter().map(Into::into);
    |                                                 ^^^^^^^^^^^^^^^^ help: consider removing
    |
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#useless_conversion
    = note: `-D clippy::useless-conversion` implied by `-D warnings`
    = help: to override `-D warnings` add `#[allow(clippy::useless_conversion)]`

Reply:
This is resolved with a recent rebase, as it was fixed in `main`